cts.json

```diff
@@ -2397,6 +2397,32 @@
         }
       ]
     },
+    {
+      "name": "filter, equals, special nothing",
+      "selector": "$.values[?length(@.a) == value($..c)]",
+      "document": {
+        "c": "cd",
+        "values": [
+          {
+            "a": "ab"
+          },
+          {
+            "c": "d"
+          },
+          {
+            "a": null
+          }
+        ]
+      },
+      "result": [
+        {
+          "c": "d"
+        },
+        {
+          "a": "null"
```

Comment:
This `"null"` needs to be JSON `null` (unquoted)

Reply:
Fixed :+1: